cmd/tikv-ctl/src/util.rs

```diff
@@ -10,6 +10,7 @@ use tikv::config::TiKvConfig;
 
 const LOG_DIR: &str = "./ctl-engine-info-log";
 
+#[allow(clippy::field_reassign_with_default)]
```

Comment:
This should be fixed.

Reply:
the original way is cleaner, and we also need to change some subfield of the field, so it needs be written in reassign anyway